pkg/util/trace.go

```diff
@@ -0,0 +1,54 @@
+package util
+
+import (
+	"context"
+
+	oc "go.opencensus.io/trace"
+	"go.opentelemetry.io/otel/api/global"
+	"go.opentelemetry.io/otel/api/propagation"
+	otel "go.opentelemetry.io/otel/api/trace"
+	"go.opentelemetry.io/otel/codes"
+	"google.golang.org/grpc/status"
+)
+
+// PropagateOtelToOpencensus arranges for spans that let us trace into libraries using opencensus.
+func PropagateOtelToOpencensus(ctx context.Context) context.Context {
```

Comment:
PropagateOpenTelemetryToOpenCensus?

Reply:
Sure, a few more characters don't make it more comical than it already is